internal/server/proto/server.proto

```diff
@@ -2134,6 +2134,8 @@ message GetJobStreamResponse {
 ********************************************************************/
 
 message Runner {
+  bool deprecated_is_odr = 4;
```

Comment:
I usually throw these on the end so I can add some comments of what these used to be.